environment_set_up/README.md

```diff
@@ -18,5 +18,14 @@ options, in increasing order of complexity and flexibility:
    server from a command shell on the HPC. Requires familiarity with the HPC command line, file
    editing, etc.
 
-If your workflow makes use of parallelism using Dask on HPC hosts, read [this](./tallgrass_dask-jobqueue.md)
```

Comment:
Should we also delete the ./tallgrass_dask-jobqueue.md file? Do we want to link to this notebook here for better understanding instead? https://github.com/hytest-org/hytest/blob/main/essential_reading/Parallel_Dask.ipynb